src/components/FoodForm.js

```diff
@@ -0,0 +1,103 @@
+import {nanoid} from 'nanoid';
+import {useState} from 'react';
+
+import {StyledDoneLabel} from './styled/StyledDoneLabel';
+import {StyledForm} from './styled/StyledForm';
+import {StyledInput} from './styled/StyledInput';
+import {StlyedList} from './styled/StyledList';
+import {StyledListItems} from './styled/StyledListItems';
+import {StyledWidthBox} from './styled/StyledWidthBox';
+
+const App = () => {
+	const [value, setValue] = useState('');
+	const [foods, setFoods] = useState([
+		{
+			id: nanoid(),
+			value: 'Bio-Müsli',
+			done: false,
+		},
+		{
+			id: nanoid(),
+			value: 'Spiegelei',
+			done: false,
+		},
+		{
+			id: nanoid(),
+			value: '1 Banane',
+			done: false,
+		},
+		{
+			id: nanoid(),
+			value: 'Walnüsse',
+			done: false,
+		},
+		{
+			id: nanoid(),
+			value: 'Franzbrötchen',
+			done: false,
+		},
+	]);
+	return (
+		<StyledWidthBox>
+			<StyledForm
+				onSubmit={event_ => {
+					event_.preventDefault();
+					setFoods([...foods, {id: nanoid(), value, done: false}]);
+					setValue('');
+				}}
+			>
+				<StyledDoneLabel>
+					What did you eat today?
+					<br />
```

Comment:
try not to use `br`and use the styling methods you know instead.

Reply:
Check